fobi_custom/plugins/pldp/form_elements/fields/single/age/forms.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+
+from fobi.base import BasePluginForm
+from pldp.forms import AGE_TYPE_CHOICES
+
+
+class PLDPAgeSingleForm(forms.Form, BasePluginForm):
+    """PLDPAgeSingleForm."""
+
+    plugin_data_fields = [
+        ("label", ""),
+        ("name", ""),
```

Comment:
I wonder if label and name should be pre-populated. It seems like the pldp plugins should be consistent, across all fields, not just the age type options. 

As for the age types, it's not entirely clear what I'm selecting. 

<img width="918" alt="screen shot 2019-02-26 at 1 24 48 pm" src="https://user-images.githubusercontent.com/13078679/53436835-10d3be00-39ca-11e9-9d54-f709c271078d.png">

We can't assume that our users have an intimate (or any knowledge) of the pldp. Is there a better way to communicate what the detail level means?

***

For the question of age, single vs. multiple ("how old are you?" vs. "how many people in this space fit into this age type?")....yes, as you said, I think we need two different form plugins. It seems like pldp was designed with the multiple input in mind. You can see how each type is a row, and the value of that row is an integer (i.e., the number of people). 

<img width="1111" alt="screen shot 2019-02-26 at 1 30 10 pm" src="https://user-images.githubusercontent.com/13078679/53437273-1d0c4b00-39cb-11e9-9392-471378684f3b.png">

I think we should build out both of these options before sharing with UCD, if you think that's possible within our given timeline. What do you think?

Reply:
I can add help text to explain what the different detail levels mean, good call. Fobi isn't set up to handle form elements with multiple inputs. I'll start digging into that to see what it would take to implement.

Reply:
"Fobi isn't set up to handle form elements with multiple inputs." Oh, shoot. You mentioned that. 

Is this a bad idea? We could have a form plugin for each age question (e.g., "How many people are 0-14"), and then group them together somehow....